### PR TITLE
OAEWDGT-180 - Use Compass for mixins

### PR DESCRIPTION
**Pull request by christianvuerings**
https://jira.sakaiproject.org/browse/OAEWDGT-180

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -70,6 +68,11 @@ end
 
 # Assets for Rails 3.1
 group :assets do
+  gem 'sass-rails', "  ~> 3.1.5"
+
+  # Use compass for get out of the box mixins
```
Comment (apex13) [inline Gemfile]: @christianv Can you remove the word 'get' here?

Comment (christianvuerings) [inline Gemfile]: @apex13 That issue should now be addressed.

File: Gemfile [hunk 2]
```diff
@@ -80,4 +82,4 @@ group :qa, :production do
   gem 'mysql2'
   gem 'backup'
   gem 'whenever'
-end
```
Comment (apex13) [inline Gemfile:83]: @christianv Can we remove this change?

Comment (christianvuerings) [inline Gemfile:83]: @apex13 That should now be reverted.

**Comment by apex13** [comment]
@christianv I'm getting the following error on the home page when using your branch:

### Sass::SyntaxError in Home#index

**Showing /home/dean/Desktop/oae/sakai-widgetlibrary/app/views/layouts/application.html.erb where line #5 raised:**

File to import not found or unreadable: compass/css3/border-radius.
Load path: Sass::Rails::Importer(/home/dean/Desktop/oae/sakai-widgetlibrary/app/assets/stylesheets/application.css.scss)
  (in /home/dean/Desktop/oae/sakai-widgetlibrary/app/assets/stylesheets/application.css.scss)

**Comment by christianvuerings** [comment]
@apex13 Did you do a `bundle install` before launching the server? It seems to work for me.
Btw I also couldn't reach you on IRC/Skype.

**Comment by apex13** [comment]
@christianv Ah yes. Okay, it's working now.

**Comment by bertpareyn** [comment]
Verified and merged via squash.

**Pull request closed without merging**